### PR TITLE
Implemented 17964: Backend Order creation Authorizenet: If invalid cr…

### PR DESCRIPTION
**Pull request by passtet**
…edit card number is entered the loader keeps spinning forever without any error message

### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title
2. ...

### Manual testing scenarios

1. ...
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @passtet. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me {$VERSION} instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/code/Magento/Sales/view/adminhtml/web/order/create/scripts.js [hunk 1]
```diff
@@ -1170,8 +1170,10 @@ define([
 
         submit : function()
         {
-            jQuery('#edit_form').trigger('processStart');
-            jQuery('#edit_form').trigger('submitOrder');
+            if(jQuery('#edit_form').valid()) {
```
Comment (williankeller) [inline app/code/Magento/Sales/view/adminhtml/web/order/create/scripts.js]: Hi @passtet, please add space between `if` and `(`. 
And what about move `jQuery('#edit_form')` to a variable? Then use it to validate and trigger methods.

**Review by williankeller — CHANGES_REQUESTED** [review]

File: app/code/Magento/Sales/view/adminhtml/web/order/create/scripts.js (hunk 1 above)
Comment (passtet) [reply app/code/Magento/Sales/view/adminhtml/web/order/create/scripts.js]: @williankeller done. I would actually refactore the whole file for those, but I guess that there would be too many conflicts :)

**Review by sidolov — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-3093](https://jira.corp.magento.com/browse/ENGCOM-3093) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
Hi @passtet. Thank you for your contribution. 
We will aim to release these changes as part of 2.2.8. 
Please check the release notes for final confirmation.

Please, consider to port this solution to 2.3 release line.
You may use [Porting tool](https://github.com/magento/magento2/wiki/Simple-way-to-port-your-solutions-automatically) to port commits automatically.

**Pull request merged**